packages/driver/src/cypress/error_messages.ts

```diff
@@ -143,28 +143,6 @@ export default {
     },
   },
 
-  selectFile: {
```

Comment:
selectFile wasn't properly alphabetized. No changes other than moving it lower in the file.